x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/alerts/user_assignment.md

```diff
@@ -0,0 +1,186 @@
+# Alert User Assignment
+
+This is a test plan for the Alert User Assignment feature
+
+Status: `in progress`. The current test plan covers functionality described in [Alert User Assignment](https://github.com/elastic/security-team/issues/2504) epic.
+
+## Useful information
+
+### Tickets
+
+- [Alert User Assignment](https://github.com/elastic/security-team/issues/2504) epic
+- [Add test coverage for Alert User Assignment](https://github.com/elastic/kibana/issues/171307)
+- [Write a test plan for Alert User Assignment](https://github.com/elastic/kibana/issues/171306)
+
+### Terminology
+
+- **Assignee**: The user assigned to an alert.
+
+- **Assignees field**: The alert's `kibana.alert.workflow_assignee_ids` field which contains an array of assignees IDs.
```

Comment:
Can we specify that the `id` corresponds to the profile endpoints user `id`?

Reply:
done